core/variant_call.cpp

```diff
@@ -1438,13 +1438,12 @@ Variant::Type Variant::get_method_return_type(Variant::Type p_type, const String
 }
 
 Vector<Variant> Variant::get_method_default_arguments(Variant::Type p_type, const StringName &p_method) {
+	ERR_FAIL_INDEX_V(p_type, Variant::VARIANT_MAX, Vector<Variant>());
 	const _VariantCall::TypeFunc &tf = _VariantCall::type_funcs[p_type];
 
 	const Map<StringName, _VariantCall::FuncData>::Element *E = tf.functions.find(p_method);
-	if (!E) {
-		return Vector<Variant>();
-	}
-
+	ERR_FAIL_COND_V(!E, Vector<Variant>());
```

Comment:
I don't know why the original code was this way but are we sure that this should always be an error condition? Or are there valid cases in which `p_method` wouldn't be found but we want to handle it gracefully instead of printing an error?

Reply:
reduz put it there on master, so decided to copy it as is